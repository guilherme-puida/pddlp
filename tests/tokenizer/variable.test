?var
?var1
?var-hy
?var__
?1 ?_ ?-
