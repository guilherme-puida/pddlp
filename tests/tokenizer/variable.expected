[01:01] PDDLP_TOKEN_VARIABLE ?var
[02:01] PDDLP_TOKEN_VARIABLE ?var1
[03:01] PDDLP_TOKEN_VARIABLE ?var-hy
[04:01] PDDLP_TOKEN_VARIABLE ?var__
[05:01] PDDLP_TOKEN_ERROR first character of a variable should be a letter
[05:04] PDDLP_TOKEN_ERROR first character of a variable should be a letter
[05:07] PDDLP_TOKEN_ERROR first character of a variable should be a letter
EOF
error count: 3